bindep.txt

```diff
@@ -0,0 +1 @@
+systemd-python
```

Comment:
According [to the documentation](https://github.com/systemd/python-systemd#installation) and my own tests, the package name is `python3-systemd`